linera-views/src/common.rs

```diff
@@ -41,6 +41,51 @@ pub(crate) enum Update<T> {
     Set(T),
 }
 
+#[derive(Clone, Debug)]
+pub(crate) struct DeletionPrefixes {
```

Comment:
maybe `DeletionSet` or `DeletionKeySet`? (singular nouns work better for containers)

Reply:
Ok for `DeletionSet`.